dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenAPI-1.1/com.ibm.websphere.appserver.mpOpenAPI-1.1.feature

```diff
@@ -44,7 +44,7 @@ IBM-SPI-Package: \
  com.ibm.ws.microprofile.openapi,\
  com.ibm.ws.microprofile.openapi.ui,\
  com.ibm.ws.microprofile.openapi.1.1.model,\
- com.ibm.ws.com.fasterxml.jackson.2.9.1
+ io.openliberty.com.fasterxml.jackson
```

Comment:
This is the old feature. Do we need to change it at all? I thought not.

Reply:
Aside from the fact that the SmallRye implementation requires the `2.11.2` version of the relevant Jackson libraries, we need to align the other versions of the feature on `2.11.2` as well.